docs-website/src/pages/slacksurvey/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import Layout from '@theme/Layout';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+
+function SlackSurvey() {
+    const { siteConfig = {} } = useDocusaurusContext();
+
+    useEffect(() => {
+        const script = document.createElement('script');
+        script.src = "//js.hsforms.net/forms/embed/v2.js";
+        script.async = true;
+        script.type = 'text/javascript';
+        document.body.appendChild(script);
+
+        script.onload = () => {
+            if (window.hbspt) {
+                window.hbspt.forms.create({
+                    region: "na1",
+                    portalId: "14552909",
+                    formId: "91357965-a8dc-4e20-875e-5f87e6b9defb",
+                    target: '#hubspotForm' // Targeting the div with the specific ID
```

Comment:
it seems to autofill john doe?
<img width="742" alt="image" src="https://github.com/datahub-project/datahub/assets/12566801/66384373-505f-4ccf-b0b5-bfd38e3e773d">